dds/DCPS/XTypes/TypeLookupService.cpp

```diff
@@ -560,6 +563,18 @@ void TypeLookupService::complete_to_dynamic_i(DynamicTypeImpl* dt,
   case TK_ENUM:
     td->kind(TK_ENUM);
     td->name(cto.enumerated_type.header.detail.type_name.c_str());
+    if (cto.enumerated_type.enum_flags & IS_FINAL) {
+      td->extensibility_kind(DDS::FINAL);
+    } else if (cto.enumerated_type.enum_flags & IS_APPENDABLE) {
+      td->extensibility_kind(DDS::APPENDABLE);
+    } else {
+      if (DCPS::log_level >= DCPS::LogLevel::Warning) {
```

Comment:
Should this be a warning?  It's expected behavior since the spec says that's how the enum_flags are used.

Reply:
What section of the spec are you referring to?

Reply:
The one that caused this problem: `typedef TypeFlag EnumTypeFlag; // Unused. No flags apply`

Reply:
So, the IDL implies that they aren't used but the rest of the document does?